### PR TITLE
Added basic scoring

### PR DESCRIPTION
**Pull request by jae-huh**
I have added basic scoring. It allows you to click a button and get a random number between 0 and 10. When you click another time, it adds the new number to the old to show you the total score.

This is for issue #2

### CODE REVIEW

**Comment by jae-huh** [comment]
Hey @ro-savage can you have a look at my PR and let me know what you think?

File: css/styles.css [hunk 1]
```diff
@@ -2,4 +2,23 @@
   font-size: 50px;
   margin: auto;
   width: 500px;
+  text-align: center;
+}
+
+#totalscore {
```
Comment (ro-savage) [inline css/styles.css]: `#score` and `#totalscore` are the same. Why not making a class called .scores and then style that?

File: index.html [hunk 2]
```diff
@@ -2,11 +2,14 @@
...
-    <script src="js/jquery.js"></script>
-    <script src="js/lodash.js"></script>
+    <script src="js/vendor/jquery.js"></script>
+    <script src="js/vendor/lodash.js"></script>
+    <script src="js/main.js"></script>
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
     <div id="score">The score</div>
+    <div id="totalscore">Total score</div>
```
Comment (ro-savage) [inline index.html]: Maybe it would be good for it to say 
'Score: [score]' 
'Total score: [totalScore]'

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+
+  $('#bowl').click(function(){
+    var score = Math.floor(Math.random() * 11);
+    changeScore(score);
+
+    totalscore = totalscore + score;
```
Comment (ro-savage) [inline js/main.js:7]: Awesome work Jae!

**Comment by ro-savage** [comment]
Just a few comments. Really good work @Jae-Huh !

**Pull request merged**